tests/actions/EnforceActionExportRestrictions.ts

```diff
@@ -12,11 +12,6 @@ import * as Task from '@userActions/Task';
 // and prevents side-effects that you may not be aware of. It also allows each file to access Onyx data in the most performant way. More context can be found in
 // https://github.com/Expensify/App/issues/27262
 describe('ReportUtils', () => {
-    it('does not export getParentReport', () => {
-        // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
-        expect(ReportUtils.getParentReport).toBeUndefined();
-    });
```

Comment:
I read the comment above but I think it's not optimal to create yet another `Onyx.connect` to fetch the same data. I can change it but I think it would be better to have only one place where we load all reports to memory.